community-operators/tidb-operator/tidb-operator.1.0.0-beta1.clusterserviceversion.yaml

```diff
@@ -0,0 +1,909 @@
+apiVersion: operators.coreos.com/v1alpha1
+kind: ClusterServiceVersion
+metadata:
+  name: tidb-operator.1.0.0-beta1
+  namespace: placeholder
+  annotations:
+    alm-examples: |-
+        [  
+          {
+            "apiVersion": "pingcap.com/v1alpha1",
+              "kind": "TidbCluster",
+              "metadata": {
+                "name": "demo",
+                "namespace": "operators"
```

Comment:
this should probably be "namespace": "openshift-operators"

Reply:
No need to include a namespace for example CRs, since OLM will overwrite them